src/lightning/fabric/loggers/logger.py

```diff
@@ -99,6 +99,7 @@ def rank_zero_experiment(fn: Callable) -> Callable:
     """Returns the real experiment on rank 0 and otherwise the _DummyExperiment."""
 
     @wraps(fn)
+    @rank_zero_only
```

Comment:
This shouldn't be here. The point of this function is that it should return a dummy experiment on rank > 0, but right now this makes it return None. 

Maybe you can change the logic to:

```py
if rank > 0
   return _DummyExperiment()
return fn(self)
```
to make dynamo happy?

Reply:
Done :+1: